lib/ten_green_bottles.rb

```diff
@@ -0,0 +1,22 @@
+require_relative "verses"
+require_relative "song"
+require_relative "display"
+
+class TenGreenBottles
+
+  def initialize(display)
+    @display = display
+  end
+
+  def run_song
+    @display.user_prompt("Enter the number of bottles")
+    @display.user_bottles
+    @display.display_song
+  end
+end
+
+verses = Verses.new
+song = Song.new(verses)
+display = Display.new(song)
+bottles = TenGreenBottles.new(display)
+bottles.run_song
```

Comment:
All of these lines 18-22 could be pulled out into a run script, but I'll show you that some other time.